src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -205,24 +233,38 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     # Send registration requests.
     registration_responses = self._sas.Registration(registration_requests)['registrationResponse']
 
-    # Check registeration response,
+    # Check registration response,
     # responseCode should be 103 (INVALID_VALUE).
     for registration_response in registration_responses:
-      self.assertEqual(registration_response['response']['responseCode'], 103)
+      self.assertEqual(registration_response['response']['responseCode'], 130)
 
     del registration_responses
 
-    # TODO
     # Step 2: Create a CPI database which includes CPI user with credentials
     # matching those used to create the request in Step 1.
+    cpi_database = DatabaseServer('CPI Database',config['cpiDatabaseConfig']['hostName'],
+                                  config['cpiDatabaseConfig']['port'])
+
+    # Start CPI database server.
+    cpi_database.start()
+
+    # Prepare the file list.
+    files = {('/' + file['public_key_file']): file['public_key_file'] for file in config['cpiDatabaseConfig']['cpis']}
```

Comment:
Please don't use the var name `file` since it is a Python keyword. Probably `cpi` makes more sense here anyway.

Reply:
Fixed as per suggestion.